### PR TITLE
Rewrite using pathlib

### PR DESCRIPTION
**Pull request by themorlan**
Not yet tested on windows!

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -11,46 +11,37 @@
...
     page = cont.load_page(0)
     page_front = fitz.open()
     page_front.insert_pdf(stat, from_page=0, to_page=0)
     page.show_pdf_page(page.rect, page_front, pno=0, keep_proportion=True, overlay=False, oc=0, rotate=0, clip=None)
 
-    # output file name and dir
-    result_file = content_file.replace(DOC_FOLDER, OUTPUT_FOLDER)
+    # check if files need 2nd page. Needs workaround with byte-decoding because pathlib uses
+    # different byte decoding of umlaut than python strings
+    if input_path.stem.startswith(b'Aufkla\xcc\x88rung_CT'.decode()):
```
Comment (luc-ass) [inline main.py]: this appends the "second page" to all `Aufklärung_CT*` and `Aufklärung_MRT`, thus appending it to `*_Punktion_Drainage` etc.

Disregard if intended

File: main.py [hunk 2]
```diff
@@ -11,46 +11,37 @@
...
 
     page = cont.load_page(0)
     page_front = fitz.open()
     page_front.insert_pdf(stat, from_page=0, to_page=0)
     page.show_pdf_page(page.rect, page_front, pno=0, keep_proportion=True, overlay=False, oc=0, rotate=0, clip=None)
 
-    # output file name and dir
-    result_file = content_file.replace(DOC_FOLDER, OUTPUT_FOLDER)
+    # check if files need 2nd page. Needs workaround with byte-decoding because pathlib uses
+    # different byte decoding of umlaut than python strings
```
Comment (luc-ass) [inline main.py]: can this be configured?

**Review by luc-ass — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (themorlan) [reply main.py]: What do you mean by configured?

File: main.py (hunk 1 above)
Comment (themorlan) [reply main.py]: Ok, get it. Needs further workaround

**Comment by themorlan** [comment]
Umlaut workaround does not work on windows. Actually not needed -_-

File: main.py (hunk 1 above)
Comment (luc-ass) [reply main.py]: > Umlaut workaround does not work on windows. Actually not needed -_-

match with regex?

Comment (luc-ass) [reply main.py]: Untested:
```py
if re.match(r"^Aufkl.rung_CT", input_path.stem)
```

**Review by luc-ass — APPROVED** [review]
LGTM

**Pull request merged**